src/ui/server/controllers/insurance/policy/map-submitted-data/policy/date-fields/index.test.ts

```diff
@@ -0,0 +1,139 @@
+import mapDateFields from '.';
+import FIELD_IDS from '../../../../../../constants/field-ids/insurance/policy';
+import createTimestampFromNumbers from '../../../../../../helpers/date/create-timestamp-from-numbers';
+
+const {
+  CONTRACT_POLICY: {
+    REQUESTED_START_DATE,
+    SINGLE: { CONTRACT_COMPLETION_DATE },
+  },
+} = FIELD_IDS;
+
+describe('controllers/insurance/policy/map-submitted-data/policy/date-fields', () => {
+  let date = new Date();
+  let day: number;
+  let month: number;
+  let year: number;
+
+  beforeEach(() => {
+    date = new Date();
+
+    day = date.getDate();
+    month = date.getMonth() + 1;
+    year = date.getFullYear();
+  });
+
+  describe(`when ${REQUESTED_START_DATE} day, month and year fields are provided`, () => {
+    it(`should return an object with ${REQUESTED_START_DATE} as a timestamp`, () => {
+      const mockBody = {
+        [`${REQUESTED_START_DATE}-day`]: '1',
```

Comment:
is there a reason this is 1 and not `day`?

Reply:
Updated to use `day` - will push once another PR is merged and this can be updated with `no-pdf`